docs/misc/NewToolDocExample.md

```diff
@@ -0,0 +1,57 @@
+# New Tool
+
+Brief description.
+This tool is awesome and does a lot of things.
+
+(Optional): provide reports, graphics, figures and other resources to better
+explain how the tool works.
+
+## Commands
+
+### command_1
+
+```
+    command_1 [-argA] [-argB] <file>
+```
+
+| Parameter | Description | Default      | Allowed values |
+|-----------|-------------|--------------|----------------|
+| `-argA`   | Argument A. | 0            | `int`          |
+| `-argB`   | Argument B. | 42           | [25-50]        |
+| `<file>`  | Input file. | **Required** | `string`       |
+
+
+## Example scripts
+
+## Regression tests
+
+| Test    | Description |
+|---------|-------------|
+| Test 1. | Argument A. |
+| Test 2. | Argument B. |
+| Test 3. | Input file. |
+
+## Limitations
+
+Things the tool cannot do.
+
+## FAQs
+
+Frequent questions about this tool in particular.
+
+Replace NEW_TOOL with the name of your tool in the link below.
+Use `%20` instead of spaces.
+
+Checkout [GitHub discussion](https://github.com/The-OpenROAD-Project/OpenROAD/discussions/categories/q-a?discussions_q=category%3AQ%26A+NEW TOOL+in%3Atitle) about this tool.
+
+## External references (Optional)
+
+- List papers, link to tools, etc
+
+## Authors (Optional)
+
+- Jane Doe.
+
+## License
+
+BSD 3-Clause License
```

Comment:
This should also apply to current tools- next pass

Reply:
Yes, this template will be applied for current and future toosl.